static/js/components/UploadResumeView_test.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import { assert } from "chai"
+import sinon from "sinon"
+import { shallow } from "enzyme"
+
+import UploadResumeView, {
+  ResumeUploader,
+  UploadResumeView as InnerUploadResumeView
+} from "./UploadResumeView"
+
+import { makeApplicationDetail } from "../factories/application"
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { Field } from "formik/dist/index"
+
+describe("UploadResumeView", () => {
+  let helper, renderPage, application
+
+  beforeEach(() => {
+    application = makeApplicationDetail()
+    helper = new IntegrationTestHelper()
+    renderPage = helper.configureHOCRenderer(
+      UploadResumeView,
+      InnerUploadResumeView,
+      {},
+      { application: application }
+    )
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  it("renders the drawer title, and other messages", async () => {
+    const { inner } = await renderPage()
+    assert.equal(
+      inner.find(".drawer-title").text(),
+      "Resume or LinkedIn Profile"
+    )
+
+    assert.equal(
+      inner.find(".uploaded-resume").text(),
+      `Your uploaded resume: ${
+        application.resume_filepath ? application.resume_filepath : ""
```

Comment:
We should make sure that we're explicitly testing when the application has a resume and when it doesn't.

Reply:
are you talking specifically about this test? or about the implementation of the component?